test_problems/rankine_democxx/rankine.cpp

```diff
@@ -26,21 +20,22 @@ void saveState(F& fluid, string name)
 
 void printStates()
 {
-    size_t nStates = states.size();
-    for (size_t n = 0; n < nStates; n++) {
-        string name = states[n];
-        printf(" %5s %10.6g %10.6g  %12.6g %12.6g %5.2g \n",
+    int nStates = states.size();
+    for (int n = 0; n < nStates; n++) {
+        std::string name = states[n];
+        printf(" %5s %10.6g %10.6g %12.6g %12.6g %5.2g \n",
```

Comment:
How does this not leave a trailing space on each line in the new blessed output file?

Reply:
Probably because I opened the blessed output file in the text editor, which automatically strips end of line whitespace. I changed the function so it doesn't include the last space to match the blessed output.